script/Arcueid/Day 6/06_00_ARC06_3_2.txt

```diff
@@ -338,6 +341,7 @@ Well then.
 [sha:8cca9cf3ec8239a1c1b66abc6ab6a86286e5dff0]{
 -- Page 300, Offset 14647.
 -- 斎木業人が片手をあげると、駐車場に待機していた乗用車が玄関前のロータリーに移動してきた。
+// to be rechecked, change rotary
```

Comment:
I think the complaint with rotary was that it's a direct katakana TL? I feel like it's fine. Or just drop it for 
> ...rolls up in front of the entrance...